shoop/simple_cms/models.py

```diff
@@ -12,13 +12,15 @@
 from django.utils.encoding import force_text, python_2_unicode_compatible
 from django.utils.timezone import now
 from django.utils.translation import ugettext_lazy as _
-from parler.managers import TranslatableQuerySet
+from mptt.managers import TreeManager
+from mptt.models import MPTTModel, TreeForeignKey
+from parler.managers import TranslatableManager
 from parler.models import TranslatableModel, TranslatedFields
 
 from shoop.core.fields import InternalIdentifierField
 
 
-class PageQuerySet(TranslatableQuerySet):
+class PageManager(TranslatableManager, TreeManager):
```

Comment:
Is there such thing as TreeQuerySet? Would like to use query set rather than manager, because queryset can be piped (`Page.objects.filter(something).visible()`).